tap-gui/plugins/scc-tap-gui.hbs.md

```diff
@@ -186,3 +186,35 @@ source code or image, including vulnerabilities from past scans.
 > On 15 January, the log4j CVE issue is found while scanning image DEF. If a user returns to the
 > **Image Scan** stage for image ABC, the log4j CVE issue appears and is associated with the
 > `log4shell` package.
+
+### Triage Vulnerabilities
+
+This functionality allows you to store an analysis for each of the vulnerabilities found in the scan.
+
+#### Enable Triage Status
+Tanzu Application Platform GUI has this functionality disabled. To use it, add it to config section within your `tap-values.yaml` file. See the following example:
```

Comment:
Change to... "The triage vulnerability feature is turned off by default in Tanzu Developer Portal.  To enable, add the following to your  `config` section within the `tap-values.yaml` file.  See the following example.